File: cpp/program33.cpp
// program48.cpp

#include<iostream>
using namespace std;

class Digit
{
    public:
        int iNo;

    Digit(int X)
    {
        iNo = X;
    }

    int CountDigit()
    {
        int iDigit = 0;
        int iCnt = 0;
        int iTemp = iNo;

        while(iTemp != 0)
        {
            iDigit = iTemp % 10;
            iCnt++;
            iTemp = iTemp / 10;
        }
        return iCnt;
    }
};

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout<<"Enter number"<<"\n";
    cin>>iValue;

    Digit dobj(iValue);

    iRet = dobj.CountDigit();
    cout<<"Count of the Digit is : "<<iRet<<"\n";

    iRet = dobj.CountDigit();
    cout << "Count of the Digit is : " << iRet << "\n";

    return 0;
}

File: c#/program33.cs
using System;


class MainClass
{
    public static int CountDigit(int No)
    {
        int count = 0;

        while(No != 0)
        {
            No % 10;
            count++;
            No = No / 10;
        }
        return count;
    }

    static void Main()
    {
        int ret = 0,value = 0;

        Console.WriteLine("Enter number :");
        value = int.Parse(Console.ReadLine());

        ret = CountDigit(value);
        Console.WriteLine("Count of the digit is : "+ret);
    }
}